float64 x
float64 y
float64 z
float64 alpha
float64 beta
float64 gamma
float64 distance
---
float64 x_prime
float64 y_prime
float64 z_prime
